JASP-Engine/JASP/R/LSbinomialtesting.R

```diff
@@ -849,12 +854,27 @@ LSbinomialtesting   <- function(jaspResults, dataset, options, state = NULL){
       tablePredictions$addColumns(0:options[["predictionN"]])
     }
     
-    if (options[["plotsPredictionPostType"]] %in% c("joint", "conditional")){
-      for(i in 1:length(options[["priors"]])){
-        tablePredictions$addColumnInfo(name = paste0("hyp_", i), title = gettextf("P(Successes|%s)", options[["priors"]][[i]]$name), type = "number")
-      }
-    } else if (options[["plotsPredictionPostType"]] == "marginal")
-      tablePredictions$addColumnInfo(name = "marginal", title = gettextf("P(Successes)"), type = "number")
+    
+    if (ready[2]){
```

Comment:
Why is `ready` a vector/list? Can it be named so that you are not indexing by number? This solution does not seem optimal.

Reply:
yeah, that's a good idea. Changed to a name vector